ddtrace/compat.py

```diff
@@ -183,6 +190,7 @@ def to_unicode(s):
 
 
 def get_connection_response(conn):
+    # type: (...) -> Response
```

Comment:
```suggestion
    # type: (httplib.HTTPConnection) -> Response
```

This should be the correct typing here

Reply:
Since `httplib.HTTPConnection` is from `six.moves.httplib`, which itself causes mypy issues relating to importing attributes, it's not trivial to add this specific connection typing into the typedef. Thoughts @Kyle-Verhoog @brettlangdon ?

Reply:
I'm ok leaving it for now